rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -27,11 +36,13 @@ class CountVectorsFeaturizer(Featurizer):
     from https://arxiv.org/abs/1810.07150.
     """
 
-    provides = ["text_features"]
+    provides = ["text_features", "intent_features", "response_features"]
 
     requires = []
 
     defaults = {
+        # whether to use a shared vocab
+        "use_shared_vocab": False,
```

Comment:
please throw a warning in the other config that used this key before you moved it so the user knows he should be moving the configuration value to this component instead

Reply:
@tmbo vocabulary sharing never existed before and hence the default value is False.
Did I get your question correctly?

Reply:
Added a warning log for intent tokenization, which existed before in embedding intent classifier and now in whitespace tokenizer